WookieeMatt/Essence20
# Armor: Adding evasion, multiple traits, migration

Addresses https://github.com/WookieeMatt/Essence20/issues/219

In this MR:
- For Armor, replacing `effect` with `bonusEvasion` and `bonusToughness`
- Armor can now have multiple `traits` instead of a single `trait`
- Adding `Computerized` and `Plating` as Armor traits
- Fixes in the Armor item sheet and defense templates to accommodate these
- Laying the foundations for doing migrations by heavily copying from dnd5e
- Migration to handle the above changes

Testing:
- 🚨 PLEASE TEST ON AN E20 WORLD YOU DON'T CARE ABOUT BREAKING 🚨
- Before pulling down this branch, ensure you have Armor items created in your world, as well as ones that belong to Actors and inside a compendium. Have a mix of Deflective and Silent traits.
- Pull down this branch, `armor-add-evasion`
- Be a pal and open devtools by pressing F12 before launching the test world. If something goes wrong, the logs could be helpful.
- Launch the test world
- You should see messages about the migration starting and eventually completing
- All Armor items that had Deflective or Silent selected should have those boxes checked
- Armor Bonus Toughness should be equivalent to whatever Effect had been
- Defenses should still calculate correctly, especially from Armor Bonus Toughness and Bonus Evasion
